### PR TITLE
[#3] [Chore] [iOS] Configure CI for the project

### LINKED ISSUE #3: [Chore] [iOS] Configure CI for the project
## Why

To improve production quality, continuous integration (CI) takes an important part in the development process.

## Acceptance criteria

Set up a CI pipeline that satisfies the development standard.

- Build application in CI
- Run unit tests
- Support linting for checking code convention

### PR DESCRIPTION
**Pull request by nmint8m**
- Close #3

## What happened 👀

In this PR, I set up the following jobs for iOS CI:

- [x] Build application in CI
- [x] Run unit tests on iOS
- [x] Support linting for checking code convention
- [ ] Run unit tests on `shared` module will be done in another PR

## Insight 📝

N/A

## Proof Of Work 📹

N/A

### CODE REVIEW

**Comment by markgravity** [comment]
@nmint8m CI failure 🙌

File: .github/workflows/ios_lint_and_test.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Generate KMM frameworks for Cocoapods
```
Comment (markgravity) [inline .github/workflows/ios_lint_and_test.yml:46]: Do we miss this step? 🤔 
```
    - name: Setup Konfig Properties
      env:
        BUILD_KONFIG_PROPERTIES: ${{ secrets.BUILD_KONFIG_PROPERTIES }}
      run: |
       touch ../buildKonfig.properties
       echo $BUILD_KONFIG_PROPERTIES | base64 --decode > ../buildKonfig.properties
```

File: buildSrc/src/main/kotlin/appPackage/BuildKonfig.example.kt [hunk 2]
```diff
@@ -1,9 +1,9 @@
-object BuildKonfig {
-    const val CLIENT_ID_PRODUCTION = ""
-    const val CLIENT_SECRET_PRODUCTION = ""
-    const val BASE_URL_PRODUCTION = ""
-
-    const val CLIENT_ID_STAGING = ""
-    const val CLIENT_SECRET_STAGING = ""
-    const val BASE_URL_STAGING = ""
-}
+//object BuildKonfig {
```
Comment (markgravity) [inline buildSrc/src/main/kotlin/appPackage/BuildKonfig.example.kt:1]: Why do we comment on these? 🙏

**Review by thiennguyen0196 — APPROVED** [review]
Apart from @markgravity 's comment, the rest LGTM

File: .github/workflows/ios_lint_and_test.yml (hunk 1 above)
Comment (nmint8m) [reply .github/workflows/ios_lint_and_test.yml:46]: @markgravity  No, I set up on the line 22-27

File: buildSrc/src/main/kotlin/appPackage/BuildKonfig.example.kt (hunk 2 above)
Comment (nmint8m) [reply buildSrc/src/main/kotlin/appPackage/BuildKonfig.example.kt]: @markgravity  I revert back to normal.

**Review by markgravity — APPROVED** [review]

**Pull request merged**